connectors/socialcast/src/main/java/com/vmware/connectors/socialcast/SocialcastController.java

```diff
@@ -98,6 +90,7 @@
  * </pre>
  */
 @RestController
+@SuppressWarnings("PMD.UseConcurrentHashMap")
```

Comment:
Won't need this when we stop checking controversial rules.

Reply:
I am ignoring this rule in pmdrules.xml. I have restored it to ```HashMap```.